assets/js/data/cart/actions.js

```diff
@@ -85,17 +85,17 @@ export function receiveCartItem( response = {} ) {
 
 /**
  * Returns an action object to indicate if the specified cart item
- * is being updated; i.e. removing, or changing quantity.
+ * quantity is being updated.
  *
+ * @param {boolean} isPendingQuantity Flag for update state; true if API request is pending.
  * @param {string} cartItemKey Cart item being updated.
- * @param {boolean} isQuantityPending Flag for update state; true if API request is pending.
  * @return {Object} Object for action.
  */
-export function itemQuantityPending( cartItemKey, isQuantityPending ) {
+export function itemPendingQuantity( isPendingQuantity, cartItemKey ) {
```

Comment:
Why change the order of the arguments?

In order for anything to happen we _require_ a cartItemKey otherwise the reducer has no clue what item to apply the "pending" state to. Convention is to put _required_ arguments first and optional arguments last (yes, I consider `isPendingQuantity` optional, and we could default it to a sensible default here - maybe true?)

So please return this to it's original order.

Reply:
I did consider both of them required, my initial problem is that if you have a flag parameter, it should be named or it should be the first  (and only flag) parameter, because we end up risking with functions like in which we have no idea what that flag means.
```js
apiRequest(
  'products',
  'GET',
  { category: 3 },
  false,
  null,
  true
)
```

granted this is not the case here and might never be the case, but possibly we should enforce a certain default in the future (if the function has 3 parameters, it should be turned to a deconstructed object, as in, named parameters).

> So please return this to it's original order.

Reply:
>  my initial problem is that if you have a flag parameter, it should be named or it should be the first (and only flag) parameter, because we end up risking with functions like in which we have no idea what that flag means.

In general I think if there's multiple arguments having different flag parameters there's a larger problem that should be dealt with (and having a defined object as the argument with named arguments is probably the better way to go).  So as you noted, the argument doesn't really apply here.

I still stand by my original comment that `cartItemKey` is more important than the flag argument and should be first.  I also would suggest that some improvements could be made to the action creator name to make it a bit clearer it is actually setting a flag for the item - maybe `itemIsPendingQuantity( cartItemKey, isPendingQuantity = true )`

Reply:
this is fixed in 0aaf2801e2bc96ba0aa3bccb981e49780bce03b9